AutoRest/Generators/Java/Java/JavaCodeNamer.cs

```diff
@@ -348,7 +348,7 @@ public static String ImportedFrom(PrimaryType primaryType)
             else if (primaryType == PrimaryType.TimeSpan ||
                 primaryType.Name == "Period")
             {
-                return "java.time.Period";
+                return "org.joda.time.Period";
```

Comment:
Sorry I might have missed some context, but what's the reason you choose `Period` over `Duration` or `Interval`? (Thumbs up for using `org.joda.time` instead!)